backend/README.md

```diff
@@ -1,11 +1,33 @@
-# backend
+# EmbassyOS Backend
+
 - Requirements:
-    - [Install Rust](https://rustup.rs)
-    - Recommended: [rust-analyzer](https://rust-analyzer.github.io/)
-    - [Docker](https://docs.docker.com/get-docker/)
-    - [Rust ARM64 Build Container](https://github.com/Start9Labs/rust-arm-builder)
-- Installation: 
-    - follow the setup instructions [here](backend/README.md)
+  - [Install Rust](https://rustup.rs)
+  - Recommended: [rust-analyzer](https://rust-analyzer.github.io/)
+  - [Docker](https://docs.docker.com/get-docker/)
+  - [Rust ARM64 Build Container](https://github.com/Start9Labs/rust-arm-builder)
 - Scripts (run withing the `./backend` directory)
-    - `build-prod.sh` - compiles a release build of the artifacts for running on ARM64
-    - `build-dev.sh` - compiles a development build of the artifacts for running on ARM64
+  - `build-prod.sh` - compiles a release build of the artifacts for running on ARM64
+  - `build-dev.sh` - compiles a development build of the artifacts for running on ARM64
+- A Linux computer or VM
+
+## Structure
+
+The EmbassyOS backend is broken up into 4 different binaries:
+
+- embassyd: This is the main workhorse of EmbassyOS - any new functionality you want will likely go here
+- embassy-init: This is the component responsible for allowing you to set up your device
+- embassy-cli: This is a CLI tool that will allow you to issue commands to embassyd and control it similarly to the UI
+- embassy-sdk: This is a CLI tool that aids in building and packaging services you wish to deploy to the Embassy
+
+Finally there is a library that supports all four of these tools.
```

Comment:
Remove this line?

Reply:
Will this be in reference to the Rust Crate?  Is that published?

Reply:
it is the `embassy` lib